### DISCUSSION TITLE
mod does not work

### DISCUSSION THREAD

**Discussion by deresara** [TEHE-Studios/ExpandedHelicopterEvents#283] [Q&A]
In short, I installed the mod and started a game, I adjusted things to my liking and started playing, 2 months have already passed in the game and there still hasn't been a single event, I don't know why that happens, obviously the mod is activated and the game detects it as a position, but it does not work

**Comment by Chuckleberry-Finn** [comment]
https://github.com/TEHE-Studios/ExpandedHelicopterEvents/issues/291
